src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -196,24 +196,34 @@
         <Grid.ColumnDefinitions>
             <ColumnDefinition MaxWidth="500"
                               MinWidth="1"
-                              Name="LibraryViewColumn">
-                <ColumnDefinition.Width>
+                              Name="LeftExtensionsViewColumn">
+                <!--<ColumnDefinition.Width>
```

Comment:
This is commented out because I found it unnecessary and lead to broken behavior, but need to run unit tests on this